.github/workflows/pkgci_test_sharktank.yml

```diff
@@ -69,6 +74,7 @@ jobs:
         run: |
           source ${VENV_DIR}/bin/activate
           python -m pip install iree-test-suites/sharktank_models/
+          python -m pip install -r iree-test-suites/sharktank_models/requirements-iree.txt
```

Comment:
Remove this. The workflow already installed dev IREE packages (testing dev packages on pull request and push events is the whole point of these workflows), but this command installs nightly packages: https://github.com/iree-org/iree-test-suites/blob/main/sharktank_models/requirements-iree.txt

Reply:
ok, but it was seg faulting without this